timesketch/lib/datastores/opensearch.py

```diff
@@ -706,8 +729,24 @@ def search(
             else:
                 cause = str(e)
 
-            es_logger.error("Unable to run search query: %s", cause, exc_info=True)
-            raise ValueError(cause) from e
+            os_logger.error(
+                "Unable to run search query. Error: %s. "
+                "Sketch ID: %s. Indices: %s. "
+                "Query String: '%s'. Query Filter: %s. "
```

Comment:
We probably shouldn't log the query string here since this will add potential case data into the logs.
Let's remove it here for now. Long term solution would probably a prometheus logging for those if we need debugging. Or add a separate `os_logger.debug` call that a TS admin can explicitly enable to debug issues.

Reply:
SGTM